benchmark/opperf/rules/default_params.py

```diff
@@ -164,4 +312,5 @@
                           "mu", "sigma", "lam", "alpha", "beta", "gamma", "k", "p",
                           "low", "high", "weight", "bias", "moving_mean", "moving_var",
                           "weight", "weight32", "grad", "mean", "var", "mom", "n", "d",
-                          "v", "z", "g", "delta", "args"]
+                          "v", "z", "g", "delta", "args", "label", "data1", "data2",
+                          "parameters", "state", "loc"]
```

Comment:
why aren't the default_data_* variables you created for separate inputs added to this list? Does it work without creating NDArray?

Reply:
Yes, those variables do not need to be manually added to the NDArray list. The way it works is that the underlying arg name ("data") is the only thing that needs to be in the list, and the mutations of that arg name ("data_*") are also interpreted as an NDArray.